### ISSUE TITLE
Time limit doesn't seem to work.

### ISSUE THREAD

**Issue by UnexpectedMushroom** [MHDante/Habitica-SitePass#3]
After paying a certain amount to access a site for a certain period, it lets you into the site, but once the time elapses the popup doesn't come back. It works fine if you are navigating to different pages within  the domain, but if you are using a site where you stay on the same page (like slither.io) it doesn't seem to trigger.

I might just be misunderstanding the way the thing works, though.

**Comment by MHDante** [comment]
You're right. I think I made this decision because I didn't want it to make me lose an email or something. I'll try to address it soon.

**Comment by TheWaler** [comment]
Hi, is this extension still under development?
If not, would you mind if I fork it to fix these issues and add a couple of features?

**Comment by MHDante** [comment]
Yep, feel free! I'll pr them in and release an update!

**Comment by TheWaler** [comment]
Awesome, looks like I have a new weekends project. :)

**Comment by ofekmiz** [comment]
Almost done fixing it, Waiting for pull requests :)

**Comment by ofekmiz** [comment]
Done :)
sent pull request #11 (comment)
